version/version.go

```diff
@@ -1,5 +1,5 @@
 package version
 
 var (
-	Version = "0.2.6"
+	Version = "0.3.0"
```

Comment:
If you're going to tag this once merged, should the CHANGELOG be updated as part of this also?

Also feel free to set the image tag in operator.yaml to the release version also, rather than master.